main.cpp

```diff
@@ -28,7 +28,19 @@ int main(){
       cout << "=" << num1*num2 << endl;
     }
     else if( input == "divide" ){
-      cout << "=" << num1/num2 << endl;
+      int remainder = num1%num2;
```

Comment:
This calculation should probably be moved into the else statement below to avoid unnecessary calculations in the divide by zero edge case.